network-api/networkapi/templates/fragments/quote.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailimages_tags %}
```

Comment:
Hi @tbrlpld , just wanna double check to see if this file is basically the same as [network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/quote.html](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/f811e29932a17e54fbd839021cf30cd4044b447d/network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/quote.html) but relocated and has the comments removed?

Reply:
Ah yes. Sorry. I mean to add that as a comment. I think because of the changes inside the file GitHub did not mark this as a move. 

The internal changes were mostly removing the `srcset` which is not needed because we already define responsive variants with the `picture` (and was invalid because of missing `sizes`).